gateway/ln-gateway/src/config.rs

```diff
@@ -6,13 +6,33 @@ use url::Url;
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct GatewayConfig {
-    /// API webserver bind address
-    pub bind_address: SocketAddr,
-    /// URL under which the API will be reachable
-    pub announce_address: Url,
-    /// webserver authentication password
-    pub password: String,
+    /* lightning configs */
+    /// Lightning RPC bind address
+    pub lnrpc_bind_address: SocketAddr,
```

Comment:
Why does the gateway config also have this bind address? What kind of server will be listening there? If we are not binding to it ourselves it should be called differently imo.